dist/common/systemd/scylla-server.service

```diff
@@ -3,6 +3,7 @@ Description=Scylla Server
 Wants=scylla-jmx.service
 Wants=scylla-housekeeping-restart.timer
 Wants=scylla-housekeeping-daily.timer
+Conflicts=abrt-ccpp.service
```

Comment:
abrt-ccpp.service runs abrt-hook-ccpp. the former was removed from abrt in v 2.14.0. see https://github.com/abrt/abrt/blob/90c3957bd644ded992a36672069cdcbe0f60b3f5/abrt.spec#L1320 and the latter was remove and the latter was remove in v 2.13.0, see https://github.com/abrt/abrt/blob/90c3957bd644ded992a36672069cdcbe0f60b3f5/CHANGELOG.md?plain=1#L154 .the `abrt-hook-ccpp` script overwrites the core dump pattern file so the coredump will be processed by its own `abrt-hook-ccpp` . the abrt upstream ditched the hack in favor of systemd-coredump.

since we still need to support el7 and el8, which come with abrt-2.10, we cannot drop this change yet. but might want to note this as a comment in code, that this only applies to abrt < 2.14 .

Reply:
Added comment "This will only requires for abrt < 2.14"